client/changelist/change.go

```diff
@@ -0,0 +1,73 @@
+package changelist
+
+import (
+	"encoding/json"
+
+	"github.com/endophage/gotuf/data"
+)
+
+type tufChange struct {
+	Actn       string `json:"action"`
+	Role       string `json:"role"`
+	ChangeType string `json:"type"`
+	ChangePath string `json:"path"`
+	Data       []byte `json:"data"`
+}
+
+func NewTufChange(action, role, changeType, changePath string, content []byte) *tufChange {
+	return &tufChange{
+		Actn:       action,
+		Role:       role,
+		ChangeType: changeType,
+		ChangePath: changePath,
+		Data:       content,
+	}
+}
+
+func NewTufTargetChange(action, target string, hash map[string]data.HexBytes, size int64, custom map[string]interface{}) (*tufChange, error) {
+	jsonCustom, err := json.Marshal(custom)
```

Comment:
This is more than a glorified case, it is a copy `// UnmarshalJSON sets *m to a copy of data.`. Why not just actually cast here, this is perfectly acceptable `json.RawMessage(jsonCustom)`